build/fix-groovy-compiler.sh

```
@@ -0,0 +1,283 @@
+\1</plugin>'''
+
+# 尝试匹配并替换
+def replace_plugin(match):
+    indent = match.group(1)
+    version = match.group(2).strip()
+    replacement = f'''{indent}<plugin>
+{indent}    <groupId>org.apache.maven.plugins</groupId>
+{indent}    <artifactId>maven-compiler-plugin</artifactId>
+{indent}    <version>{version}</version>
+{indent}    <configuration>
+{indent}        <source>${{project.java.version}}</source>
```

Shellcheck (0.11.0)

[error] Parameter expansions can't start with {. Double check syntax.

(SC2296)